oc-chef-pedant/lib/pedant/platform.rb

```diff
@@ -195,18 +195,18 @@ def create_client(name, org = self.test_org)
       payload = { "name" => clientname }
 
       if server_api_version == 0
-        r = post(api_url('/clients'), org.validator, :payload => payload)
+        r = post(api_url('/clients', org), org.validator, :payload => payload)
```

Comment:
Is it safe to change the requests for old api versions?  Does that mean we're changing the contract on people?